.rubocop_rspec_base.yml

```diff
@@ -0,0 +1,316 @@
+# This file was generated on 2022-01-10T22:24:10+00:00 from the rspec-dev repo.
+# DO NOT modify it by hand as your changes will get lost the next time it is generated.
+
+# This file contains defaults for RSpec projects. Individual projects
+# can customize by inheriting this file and overriding particular settings.
+
+Layout/AccessModifierIndentation:
+  Enabled: false
+
+# "Use alias_method instead of alias"
+# We're fine with `alias`.
+Style/Alias:
+  Enabled: false
+
+# "Avoid the use of the case equality operator ==="
+# We prefer using `Class#===` over `Object#is_a?` because `Class#===`
+# is less likely to be monkey patched than `is_a?` on a user object.
+Style/CaseEquality:
+  Enabled: false
+
+# Warns when the class is excessively long.
+Metrics/ClassLength:
+  Max: 100
+
+Style/CollectionMethods:
+  PreferredMethods:
+    reduce: 'inject'
+
+# Over time we'd like to get this down, but this is what we're at now.
+Metrics/CyclomaticComplexity:
+  Max: 10
+
+# We use YARD to enforce documentation. It works better than rubocop's
+# enforcement...rubocop complains about the places we re-open
+# `RSpec::Expectations` and `RSpec::Matchers` w/o having doc commments.
+Style/Documentation:
+  Enabled: false
+
+# We still support 1.8.7 which requires trailing dots
```

Comment:
We don't really in `rspec-rails`, it's just the 3.9 of other `rspec-*` repos.

Reply:
Fair enough.  And we don't have to use the same config.

That said, I just think it makes sense to use a consistent .rubocop_rspec_base.yml across the rspec codebases.  Same with the Hash rocket syntax changes - not needed here, but still needed for the other repos which support earlier Ruby versions.

Reply:
Makes total sense to me. When we release RSpec 4, we could update those base rules for other RSpec repos and align then with `rspec-rails` 👍 
Let's keep this base file 👍